### ISSUE TITLE
review heuristic task

### ISSUE THREAD

**Issue by nathandunn** [ucscXena/XenaGoWidget#321]
- feedback:  https://docs.google.com/document/d/14WB4Omtz2F9afg0fcWZESa0obkftFvUMjv-9GSyAdk4/edit
- other tasks: https://docs.google.com/document/d/1cP1kL08zHoimkGDV1in4zUy-EFtiBdnmCGMJ9wM14Z0/edit

**Comment by nathandunn** [comment]
"other tasks":


Delivered: https://docs.google.com/document/d/1Ic2nMF0L3TXCOxbTPEqhJ-UdYJBHLQswkemtIfg0t48/edit?ts=5d53476b

Tool:  https://xenageneset.berkeleybop.io/xena/

You are a genomic scientists in a clinical lab 

Task 1:
After sequencing cells from a number of patients with Breast Cancer you find 3 genes of interest: A, B, C

You wish to establish the relevant pathways of those genes and if they are over-represented in others.  You can use the published data in order to bolster your own results and suggests other areas of research.


After sequencing cells from a number of patients with Breast Cancer you find certain genes of interest that were frequently altered: PIK3CA, AKT3, PTEN, FADD, DEPTOR, BRCA2

You wish to establish which pathways these genes are in and to determine what other genes that are frequently altered, either by mutation or copy number statuses, in TCGA BRCA.  You will also want to check these pathways in all other cohorts including Ovarian Cancer, which tends to have related dysregulated pathways, to see if you see similar patterns, which could suggest new cancer types to look at these genes in. While scanning cohorts, you also wish quickly assess if these genes are frequently altered  in other pathways (gene sets).  

You can use this published data in order to bolster your own results and suggests other areas of research.


Task 2:
You are looking for likely genes involved in a pathogenic pathways based on other cohort data.  You will use the results to narrow down potential pathways, identify important genes in that pathway, and possibly generate new pathways / gene sets.

Task 3:
We have evidence for a likely pathway for a type of cancer.   We would like to explore that gene set in the context of known relevant cohorts as well as unexplored cohorts.  We would like to enter a modified gene set and explore across multiple cohorts.


Familiar Metaphors and Language:
Clean Functional Design: 
Freedom:
Flexibility: 

Error prevention
Even better than good error messages is a careful design which prevents a problem from occurring in the first place. Either eliminate error-prone conditions or check for them and present users with a confirmation option before they commit to the action.

Recognition rather than recall
Minimize the user's memory load by making objects, actions, and options visible. The user should not have to remember information from one part of the dialogue to another. Instructions for use of the system should be visible or easily retrievable whenever appropriate.

Flexibility and efficiency of use
Accelerators — unseen by the novice user — may often speed up the interaction for the expert user such that the system can cater to both inexperienced and experienced users. Allow users to tailor frequent actions.

Aesthetic and minimalist design
Dialogues should not contain information which is irrelevant or rarely needed. Every extra unit of information in a dialogue competes with the relevant units of information and diminishes their relative visibility.

Help users recognize, diagnose, and recover from errors
Error messages should be expressed in plain language (no codes), precisely indicate the problem, and constructively suggest a solution.

Help and documentation
Even though it is better if the system can be used without documentation, it may be necessary to provide help and documentation. Any such information should be easy to search, focused on the user's task, list concrete steps to be carried out, and not be too large.

**Comment by nathandunn** [comment]
Feedback:


Tool:  https://xenageneset.berkeleybop.io/xena/

You are a genomic scientists in a clinical lab. 

Known bug: You have to scroll over the gene view in order for the gene diffs to be updated correctly.


Task 1:
You are trying to develop a drug for Basal Head and Neck Cancer. There has been a lot of research done on Head and Neck Cancer in general, so you are looking for which pathways and genes you might target specifically for the basal subtype that may not have previously been looked at for Head and Neck Cancer in general. You are looking for pathways and genes that are altered, either via copy number changes or mutations.


Matt:
https://docs.google.com/document/d/1hVIvsFcOftRkFfvmEq7kaHd3-2lPXEoC095XqZ5NdjE/edit#
What can I interact with? 
Maybe clicking on genes should highlight other gene sets
What are green and pink (and clicking), but maybe that should be a popup
The highlight of green is inconsistent with the other highlight
Two columns for each gene is VERY confusing, and hard to view as well.   Maybe having the different effects would be good. 
Its confusing to figure out the label.

Nathan: https://docs.google.com/document/d/18-nWs38bfNKKVb4cqXEVSg_Ipt_ynzUWf5-eDNqTj4k/edit#

Mary:
What do colors mean?
https://docs.google.com/document/d/1dzMKG2V710Wk9YWxUQRd53csXmIkMHSe9E-WTatxqxo


Duncan:
Bug: Crosshairs over the select piece
Inconsistent with filters vs select
Add a legend for the differentials, and green vs pink for the RHS
Label it legend

**Comment by nathandunn** [comment]
Matt feedbac:

https://docs.google.com/document/d/18-nWs38bfNKKVb4cqXEVSg_Ipt_ynzUWf5-eDNqTj4k/edit#

**Comment by nathandunn** [comment]
Matt feedback:

Consistency
The cross hairs are not what I expect to see when things are clickable/interactable. It took me a little while to recognize that ‘select cohort A’ was something I could interact with.

Not sure I understand the plot on the right. I’ve never seen anything like it before. Maybe it’s because I’m not really in the cancer genomics field? Additionally, it took me a while to recognize that there was a bottom half to the plot. On my small screen all of that was ‘below the fold’ and so I didn’t realize it was there. Also, don’t really understand what the little green boxes/bars mean?
Use familiar metaphors and language

Clean and functional design
The cross hairs are distracting.

Freedom
Not super clear what I can do or interact with.
Flexibility

Recognition over recall

Show status
Thing are clickable that seem like they should not be because clicking them does nothing. 

I like that when I change cohorts or when I select some subtypes, there’s something telling me what’s happening. 
Prevent errors
The big crosshairs and my little cursor got out of sync somehow. Had to keep reminding myself where to look as I kept wanting to look at the other set of crosshairs.
Support error recovery

Provide useful help

**Comment by nathandunn** [comment]
Mary's notes: Consistency

Use familiar metaphors and language
When you hover over a gene, there is a green line around it. In other parts of the website that means it is clickable. But here nothing happens when you click
Clean and functional design
Didn't realize that arrows on either side of the cohort A or B thing were going to do anything … Thought they were just there for show
Freedom

Flexibility

Recognition over recall
Don't understand coloring. What is blue, red, green? 
Why are gene names colored the way they are?
Why does every gene have two columns of data?

What are the numbers in parentheses after the CNV+Mutation?

When hovering over the pathway names, what does the score mean?
Show status
I got it once I clicked but I didn't realize the cohort name was going to let me change it. When you click 'select cohort a', nothing happened. Need to show that this is a drop down

What is the filter (##)?
Prevent errors

Support error recovery

Provide useful help
